posthog/celery.py

```diff
@@ -382,715 +148,8 @@ def retry_signal_handler(sender, **kwargs):
-def verify_persons_data_in_sync():
-    from posthog.tasks.verify_persons_data_in_sync import (
-        verify_persons_data_in_sync as verify,
-    )
-
-    if not is_cloud():
-        return
-
-    verify()
-
-
-def recompute_materialized_columns_enabled() -> bool:
-    from posthog.models.instance_setting import get_instance_setting
-
-    if get_instance_setting("MATERIALIZED_COLUMNS_ENABLED") and get_instance_setting(
-        "COMPUTE_MATERIALIZED_COLUMNS_ENABLED"
-    ):
-        return True
-    return False
-
-
-@app.task(ignore_result=True)
-def clickhouse_materialize_columns():
-    if recompute_materialized_columns_enabled():
-        try:
-            from ee.clickhouse.materialized_columns.analyze import (
-                materialize_properties_task,
-            )
-        except ImportError:
-            pass
-        else:
-            materialize_properties_task()
-
-
-@app.task(ignore_result=True)
-def clickhouse_mark_all_materialized():
-    if recompute_materialized_columns_enabled():
-        try:
-            from ee.tasks.materialized_columns import mark_all_materialized
-        except ImportError:
-            pass
-        else:
-            mark_all_materialized()
-
-
-@app.task(ignore_result=True)
-def send_org_usage_reports():
-    from posthog.tasks.usage_report import send_all_org_usage_reports
-
-    send_all_org_usage_reports.delay()
-
-
-@app.task(ignore_result=True)
-def update_quota_limiting():
-    try:
-        from ee.billing.quota_limiting import update_all_org_billing_quotas
-
-        update_all_org_billing_quotas()
-    except ImportError:
-        pass
-
-
-@app.task(ignore_result=True)
-def schedule_all_subscriptions():
-    try:
-        from ee.tasks.subscriptions import (
-            schedule_all_subscriptions as _schedule_all_subscriptions,
-        )
-    except ImportError:
-        pass
-    else:
-        _schedule_all_subscriptions()
-
-
-@app.task(ignore_result=True, retries=3)
-def clickhouse_send_license_usage():
-    try:
-        if not is_cloud():
-            from ee.tasks.send_license_usage import send_license_usage
-
-            send_license_usage()
-    except ImportError:
-        pass
-
-
-@app.task(ignore_result=True)
-def check_flags_to_rollback():
-    try:
-        from ee.tasks.auto_rollback_feature_flag import check_flags_to_rollback
-
-        check_flags_to_rollback()
-    except ImportError:
-        pass
-
-
-@app.task(ignore_result=True)
-def ee_persist_single_recording(id: str, team_id: int):
-    try:
-        from ee.session_recordings.persistence_tasks import persist_single_recording
-
-        persist_single_recording(id, team_id)
-    except ImportError:
-        pass
-
-
-@app.task(ignore_result=True)
-def ee_persist_finished_recordings():
-    try:
-        from ee.session_recordings.persistence_tasks import persist_finished_recordings
-    except ImportError:
-        pass
-    else:
-        persist_finished_recordings()
-
+@app.on_after_finalize.connect
```

Comment:
The big win here is now we don't have to ever worry about imports!

Reply:
I'm certain that's a source of unknown bugs so that's massive